src/spikeinterface/extractors/cellexplorersortingextractor.py

```diff
@@ -118,7 +118,7 @@ def __init__(
         spike_times = spikes_data["times"]
 
         # CellExplorer reports spike times in units seconds; SpikeExtractors uses time units of sampling frames
-        unit_ids = unit_ids[:].tolist()
+        unit_ids = unit_ids[:].astype(int).tolist()
```

Comment:
Should we do strings to start moving in that direction?
```suggestion
        unit_ids = unit_ids[:].astype(int).tolist()
        unit_ids = [str(unit_id) for unit_id in unit_ids]
```